specification/subscription/resource-manager/readme.md

```diff
@@ -0,0 +1,70 @@
+# Subscription
+    
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Subscription.
+
+
+
+---
+## Getting Started 
+To build the SDK for Subscription, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information 
+These are the global settings for the Subscription API.
+
+``` yaml
+openapi-type: arm
+```
+
+``` yaml $(package-subscriptionDefinitions)
```

Comment:
since this readme.md seems to only contain one RP (subscriptions) you shouldn't need this guard  - please take a look at another readme.md file and similar a similar format https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/compute/resource-manager/readme.md

Reply:
Updated.